entrypoint.sh

```diff
@@ -22,6 +23,18 @@ GetCurrentBranch () {
     echo "$(git rev-parse --abbrev-ref HEAD)"
 }
 
+PrepareGitHubPagesDirectory() {
+	DESTINATIONDIR=$1
+	
+    # Remove all the files in GitHub Pages directory (if the directory exists)
+	if [ -d "$DESTINATIONDIR" ]; then
+        git rm -rf "$DESTINATIONDIR"
+    fi
+    
+    # Create the GitHub Pages directory if it does not exist
+    mkdir -p "$DESTINATIONDIR"
```

Comment:
I guess it is better to revert this change to the previous if-else-fi manner. In the current manner the `mkdir` will be execute in any case and `-p` is now saving it from raising an error if the directory exist.
If somebody at the first glance overlooks the `-p` argument or does not know about it, the code block becomes confusing. However, according to me, the previous manner was more readable.

Reply:
Sorry for the late response, I was quite busy in the last few days. I've added the `-p` argument because otherwise, the directory would not be created recursively (and mkdir would fail if parent directories did not exist). The previous if-else-fi statement was wrong, because `git rm -rf` will not only remove the files in the directory, but also the directory itself. After that, it has to be recreated.

Reply:
No need to be sorry :) Thanks for your clear explanation. I got your point now.